README.md

```diff
@@ -17,29 +17,59 @@ limitations under the License.
 
 [<img src="https://geode.apache.org/img/apache_geode_logo.png" align="center"/>](http://geode.apache.org)
 
-[![Build Status](https://travis-ci.org/apache/geode-examples.svg?branch=develop)](https://travis-ci.org/apache/geode-examples) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) 
+[![Build Status](https://travis-ci.org/apache/geode-examples.svg?branch=develop)](https://travis-ci.org/apache/geode-examples) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
 # Apache Geode examples
 
-This is the home of Apache Geode examples that are bundled with the project. Contributions<sup>[2]</sup> and corrections are welcome. Please talk to us about your suggestions at [dev@geode.apache.org](mailto:dev@geode.apache.org) or submit a [pull request](https://github.com/apache/geode/pull/new/develop).
+This is the home of Apache Geode examples that are bundled with the project.
+Contributions<sup>[2]</sup> and corrections are welcome. Please talk to us
+about your suggestions at [dev@geode.apache.org](mailto:dev@geode.apache.org)
+or submit a [pull request](https://github.com/apache/geode/pull/new/develop).
 
-## Example requirements
+## Adding a new example
 
-All examples:
+Follow this approach to add a new example:
 
-*  Need to be testable. Use unit tests, integration tests or whatever is applicable. Tests will run through the project's CI.
```

Comment:
Here's my thesis:  `gradle runAll` is the integration test.  

The model for building an example is prescriptive.  I want to be able to run the example and any tests from my IDE.  I want to show users how to use `gfsh` to start servers.  Wherever we can I want to distill an example into a single pane of code.

We should write unit tests as needed while being careful that those tests don't get in the way of learning.  IMO the two examples we have today don't need additional unit tests.  Do you agree?

Feedback welcome!